packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/CustomizeColumnsModal.js

```diff
@@ -76,12 +80,29 @@ const CustomizeColumnsModal = ({
     }
   };
 
+  const getVisibleColumnsCount = useCallback(() => {
+    let result = 0;
+
+    columnObjects.forEach((col) => {
+      if (col.isVisible) {
+        result++;
+      }
+    });
+    return result;
+  }, [columnObjects]);
+
   const string = searchText.trim().toLowerCase();
```

Comment:
You may simplify this by something like:

```
columnObjects.filter((col) => col.isVisible).length
```

Reply:
Thanks for the suggestion. Replaced the code block with `columnObjects.filter((col) => col.isVisible).length`